tests/test_enums.py

```diff
@@ -0,0 +1,71 @@
+import inspect
+import re
+from pathlib import Path
+from typing import List, Optional, TypeVar
+
+from bo4e.enum import anrede
+from bo4e.enum.strenum import StrEnum
+
+
+class TestEnums:
+    """
+    A test class that checks the enum construction.
+    """
+
+    starts_with_whitespace_pattern = re.compile(r"^[\s\n]+")
+    ends_with_whitespace_pattern = re.compile(r"[\s\n]+$")
+
+    TEnum = TypeVar("TEnum", bound=StrEnum)
+
+    @staticmethod
+    def _all_enum_classes() -> List[TEnum]:
+        """
+        returns a list of all bo4e.enum classes
+        """
+        arbitrary_enum_module_path = Path(anrede.__file__)
+        result = []
+        for python_file in arbitrary_enum_module_path.parent.glob("*.py"):
+            # don't ask me why. but it works.
+            enum_module = __import__("bo4e.enum." + python_file.name.split(".")[0])
+            for name, member in inspect.getmembers(enum_module.enum):
```

Comment:
```suggestion
            for _, member in inspect.getmembers(enum_module.enum):
```
`name` is not used in this for loop